binding/web/tests/factory/selenium_test.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python3
+
+import os
+import sys
+import threading
+import time
+from argparse import ArgumentParser
+from http.server import HTTPServer, SimpleHTTPRequestHandler
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+
+
+def simple_http_server(host='localhost', port=4001, path='.'):
```

Comment:
Can you please restructure this like the way the [Python mic demo for Porcupine](https://github.com/Picovoice/porcupine/blob/master/demo/python/porcupine_demo_mic.py#L23) is written? It's cleaner and more OO.